keystonejs/keystone
# Add float field type for keystone-next

I created a `float` field for Keystone-next, based on code from integer field type with some modifications. Please review it and comment if something done wrong.
If all is done right, I will try to implement `double` field type via same way.

How to test:
1. In `examples-next/basic/schema.ts` add the float field type, for example, to "Posts" list:
```patch
   Post: list({
     fields: {
       ...
+      float: float(),
        ...
     }
   }),
```

2. Try to create, update, remove items in "Posts" list - all should work well.

Tested with mongoose and knex adapters.
